utils/utils.go

```diff
@@ -0,0 +1,129 @@
+package utils
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"runtime"
+)
+
+// Convenience method to decode base64 encoded data
+func b64Decode(encoded []byte) ([]byte, error) {
+	// Introduced in v0.7 - we base64 wrap all raw data now, so we have to
+	// attempt to decode. This will return the error object if it fails
+	// and should only be invoked when encoding is set to base64
+	decoded, err := base64.StdEncoding.DecodeString(string(encoded))
+
+	return decoded, err
+}
```

Comment:
This could be written as 
```go
// Convenience method to decode base64 encoded data
func b64Decode(encoded []byte) ([]byte, error) {
	// Introduced in v0.7 - we base64 wrap all raw data now, so we have to
	// attempt to decode. This will return the error object if it fails
	// and should only be invoked when encoding is set to base64
	return base64.StdEncoding.DecodeString(string(encoded))
}
```

at which point, there isn't really a difference from just calling `base64.StdEncoding.DecodeString(string(encoded))` directly